lib/wiki_pageviews.rb

```diff
@@ -101,13 +101,23 @@ def parse_results(response)
     return data['items'] if data['items']
     # As of October 2017, the data type is https://www.mediawiki.org/wiki/HyperSwitch/errors/not_found
     return no_results if %r{errors/not_found}.match?(data['type'])
```

Comment:
Are there still also requests that return this error type, or did all 404's become `about:blank`?

Reply:
Maybe it would be better to just rely on the 404 status and ignore `type` altogether? I imagine that any case of a 404 should be treated the same way by the Dashboard, and it looks like that's the only error mode that we've ever handled anyway.

Reply:
hmm good point. I wasn't able to reproduce any request returning `https://www.mediawiki.org/wiki/HyperSwitch/errors/not_found` as `type`. If I understand correctly, [this request](https://wikimedia.org/api/rest_v1/metrics/pageviews/per-article/en.wikipedia/all-access/user/Selfie/daily/2015010100/2015010200) should be one of those that returned that type. So I think that your point about only relying on the 404 status makes total sense. Done on 41a142faab1455171a62a114fd1b9c8a63fb8381. Two specs are failing but I guess those are flaky tests?